PyChop/PyChopGui.py

```diff
@@ -749,6 +777,7 @@ def drawLayout(self):
             ['pair', 'hide', 'Pulse remover chopper freq', 'combo', '', self.setFreq, 'PulseRemoverCombo'],
             ['pair', 'show', 'Ei', 'edit', '', self.setEi, 'EiEdit'],
             ['pair', 'hide', 'Chopper 2 phase delay time', 'edit', '5', self.setFreq, 'Chopper2Phase'],
+            ['pair', 'hide', 'S2', 'edit', '', self.setS2, 'S2Edit'],
```

Comment:
Do you want to have something more descriptive than `S2`? Maybe `Detector bank center 2theta`?